diff -Naur orig/simulavr-1.0.0/configure next/simulavr-1.0.0/configure
--- orig.simulavr-1.0.0/configure	2012-02-12 16:26:50.000000000 +0100
+++ next.simulavr-1.0.0/configure	2016-10-02 11:33:19.753632379 +0200
@@ -14588,7 +14588,7 @@
                 { $as_echo "$as_me:${as_lineno-$LINENO}: WARNING: cannot find 'swig' program. You should look at http://www.swig.org" >&5
 $as_echo "$as_me: WARNING: cannot find 'swig' program. You should look at http://www.swig.org" >&2;}
                 SWIG='echo "Error: SWIG is not installed. You should look at http://www.swig.org" ; false'
-        elif test -n "1.3.18" ; then
+        elif test -n "3.0.0" ; then
                 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for SWIG version" >&5
 $as_echo_n "checking for SWIG version... " >&6; }
                 swig_version=`$SWIG -version 2>&1 | grep 'SWIG Version' | sed 's/.*\([0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*\).*/\1/g'`
@@ -14596,7 +14596,7 @@
 $as_echo "$swig_version" >&6; }
                 if test -n "$swig_version" ; then
                         # Calculate the required version number components
-                        required=1.3.18
+                        required=3.0.0
                         required_major=`echo $required | sed 's/[^0-9].*//'`
                         if test -z "$required_major" ; then
                                 required_major=0
@@ -14630,9 +14630,9 @@
                         if test $available_major -ne $required_major \
                                 -o $available_minor -ne $required_minor \
                                 -o $available_patch -lt $required_patch ; then
-                                { $as_echo "$as_me:${as_lineno-$LINENO}: WARNING: SWIG version >= 1.3.18 is required.  You have $swig_version.  You should look at http://www.swig.org" >&5
-$as_echo "$as_me: WARNING: SWIG version >= 1.3.18 is required.  You have $swig_version.  You should look at http://www.swig.org" >&2;}
-                                SWIG='echo "Error: SWIG version >= 1.3.18 is required.  You have '"$swig_version"'.  You should look at http://www.swig.org" ; false'
+                                { $as_echo "$as_me:${as_lineno-$LINENO}: WARNING: SWIG version >= 3.0.0 is required.  You have $swig_version.  You should look at http://www.swig.org" >&5
+$as_echo "$as_me: WARNING: SWIG version >= 3.0.0 is required.  You have $swig_version.  You should look at http://www.swig.org" >&2;}
+                                SWIG='echo "Error: SWIG version >= 3.0.0 is required.  You have '"$swig_version"'.  You should look at http://www.swig.org" ; false'
                         else
                                 { $as_echo "$as_me:${as_lineno-$LINENO}: SWIG executable is '$SWIG'" >&5
 $as_echo "$as_me: SWIG executable is '$SWIG'" >&6;}
diff -Naur orig/simulavr-1.0.0/configure.ac next/simulavr-1.0.0/configure.ac
--- orig.simulavr-1.0.0/configure.ac	2012-02-12 16:26:38.000000000 +0100
+++ next.simulavr-1.0.0/configure.ac	2016-10-02 11:33:19.757632430 +0200
@@ -27,7 +27,7 @@
 LT_INIT
 LT_PREREQ([2.2])
 AC_PROG_CXX
-AC_PROG_SWIG(1.3.18)
+AC_PROG_SWIG(3.0.0)
 SWIG_ENABLE_CXX
 SWIG_MULTI_MODULE_SUPPORT
 AM_CONDITIONAL([USE_SWIG],[test "x$SWIG" != 'x'])
