diff -Naur orig.simulavr-1.0.0/examples/atmega128_timer/main.c next.simulavr-1.0.0/examples/atmega128_timer/main.c
--- orig.simulavr-1.0.0/examples/atmega128_timer/main.c	2012-02-12 16:26:38.000000000 +0100
+++ next.simulavr-1.0.0/examples/atmega128_timer/main.c	2016-10-02 19:15:08.150233294 +0200
@@ -5,7 +5,7 @@
 volatile int timer2_ticks;
 
 /* Every ~ms */
-ISR(SIG_OUTPUT_COMPARE2)
+ISR(TIMER2_COMP_vect)
 {                               /* Every ~1ms */
    timer2_ticks++;
 }
diff -Naur orig.simulavr-1.0.0/examples/atmel_key/StdDefs.h next.simulavr-1.0.0/examples/atmel_key/StdDefs.h
--- orig.simulavr-1.0.0/examples/atmel_key/StdDefs.h	2012-02-12 16:26:38.000000000 +0100
+++ next.simulavr-1.0.0/examples/atmel_key/StdDefs.h	2016-10-02 19:54:12.420142464 +0200
@@ -63,8 +63,8 @@
    #define UART_BAUD_REG_H      UBRR0H
    #define UART_CONTROL_REG     UCSR0B
    #define UART_STATUS_REG      UCSR0A
-   #define UART_RCV_INT_VECTOR  SIG_UART0_RECV
-   #define UART_REG_EMPTY_INT_VECTOR  SIG_UART0_DATA
+   #define UART_RCV_INT_VECTOR  UART0_RX_vect
+   #define UART_REG_EMPTY_INT_VECTOR  UART0_UDRE_vect
    #define UART_DATA_REG        UDR0
 #elif DEVICE_NAME == ATMEGA163
    #define UART_BAUD_REG        UBRR
diff -Naur orig.simulavr-1.0.0/examples/python/ex_pinout.c next.simulavr-1.0.0/examples/python/ex_pinout.c
--- orig.simulavr-1.0.0/examples/python/ex_pinout.c	2012-02-12 16:26:38.000000000 +0100
+++ next.simulavr-1.0.0/examples/python/ex_pinout.c	2016-10-02 19:15:08.150233294 +0200
@@ -3,7 +3,7 @@
 
 volatile int timer2_ticks;
 
-ISR(SIG_OUTPUT_COMPARE2) {
+ISR(TIMER2_COMP_vect) {
    timer2_ticks++;
 }
 
diff -Naur orig.simulavr-1.0.0/examples/python/example.c next.simulavr-1.0.0/examples/python/example.c
--- orig.simulavr-1.0.0/examples/python/example.c	2012-02-12 16:26:38.000000000 +0100
+++ next.simulavr-1.0.0/examples/python/example.c	2016-10-02 19:15:08.150233294 +0200
@@ -4,7 +4,7 @@
 volatile int timer2_ticks;
 
 /* Every ~ms */
-ISR(SIG_OUTPUT_COMPARE2) {
+ISR(TIMER2_COMP_vect) {
    timer2_ticks++;
 }
 
diff -Naur orig.simulavr-1.0.0/examples/python/example_io.c next.simulavr-1.0.0/examples/python/example_io.c
--- orig.simulavr-1.0.0/examples/python/example_io.c	2012-02-12 16:26:38.000000000 +0100
+++ next.simulavr-1.0.0/examples/python/example_io.c	2016-10-02 19:15:08.150233294 +0200
@@ -5,7 +5,7 @@
 volatile int port_val;
 volatile int port_cnt;
 
-ISR(SIG_OUTPUT_COMPARE2) {
+ISR(TIMER2_COMP_vect) {
    timer2_ticks++;
 }
 
diff -Naur orig.simulavr-1.0.0/regress/timertest/timer_16bit.c next.simulavr-1.0.0/regress/timertest/timer_16bit.c
--- orig.simulavr-1.0.0/regress/timertest/timer_16bit.c	2012-02-12 16:26:38.000000000 +0100
+++ next.simulavr-1.0.0/regress/timertest/timer_16bit.c	2016-10-02 19:16:26.879222169 +0200
@@ -7,7 +7,7 @@
 ISR(SIG_OVERFLOW3) {
    timer_ticks++;
 }
-ISR(SIG_OUTPUT_COMPARE3B) {
+ISR(TIMER3_COMPB_vect) {
    timer_ticks++;
 }
 #else
diff -Naur orig.simulavr-1.0.0/regress/timertest/timer_8bit_ctc.c next.simulavr-1.0.0/regress/timertest/timer_8bit_ctc.c
--- orig.simulavr-1.0.0/regress/timertest/timer_8bit_ctc.c	2012-02-12 16:26:38.000000000 +0100
+++ next.simulavr-1.0.0/regress/timertest/timer_8bit_ctc.c	2016-10-02 19:15:08.154233344 +0200
@@ -4,11 +4,11 @@
 volatile int timer_ticks;
 
 #ifdef PROC_atmega48
-ISR(SIG_OUTPUT_COMPARE2A) {
+ISR(TIMER2_COMP_vectA) {
    timer_ticks++;
 }
 #else
-ISR(SIG_OUTPUT_COMPARE2) {
+ISR(TIMER2_COMP_vect) {
    timer_ticks++;
 }
 #endif
